### ISSUE TITLE
Escape is lost in RegEx string

### ISSUE THREAD

**Issue by AndersMad** [trullock/NUglify#286]
**This:**
```js
new RegExp(`<[\\w-\\d]$`, "i");
```

**Becomes this in source:**
```js
new RegExp(`<[\w-\d]$`,"i");
```

Note the `\\w` to `\w` - bug is related to the template literal syntax.. 

EDIT: Turns out its the same as https://github.com/trullock/NUglify/issues/245#issue-862625088

**Comment by trullock** [comment]
How are you invoking NUglify with this snippet?

**Comment by AndersMad** [comment]
```cs
result = Uglify.Js(content, new CodeSettings {
	EvalTreatment = EvalTreatment.MakeImmediateSafe,
	TermSemicolons = true
});
```

**Comment by trullock** [comment]
Ok I'll take a look

**Comment by AndersMad** [comment]
I updated the main as I grabbed the source from the error console that shows `\\w` and a final `w` - possible because its an invalid escaping..

**Comment by trullock** [comment]
I've seen this before, thought it was fixed. You on the latest version or some ancient bundler-minifier included one?

**Comment by AndersMad** [comment]
I am on `1.17.2`

**Comment by AndersMad** [comment]
Uhm - ok; My initial repor is not correct so it would seem (to my defense the power keeps failing here this late night - think its the outdoor xmas lights + an inside USB charger.. 2 x error @ same time! mental 😬)

Test case: 

```js
new RegExp(`<[\\w-\\d]$`, "i"); // FAIL
new RegExp("<[\\w-\\d]$", "i"); // OK
new RegExp(`<[\\w\\d]$`, "i"); // BAD
new RegExp(`<[\\w-\\w]$`, "i"); // BAD
new RegExp(`<[\\d-\\d]$`, "i"); // BAD
```

So FAIL is the only one that yields an RegEx exception - but BAD is also wrong.. This is the generated source:

```js
new RegExp(`<[\w-\d]$`,"i");
new RegExp("<[\\w-\\d]$","i");
new RegExp(`<[\w\d]$`,"i");
new RegExp(`<[\w-\w]$`,"i");
new RegExp(`<[\d-\d]$`,"i");
```

TL;DR: It's all about the ``` ` ``` strings - as last time I reported something like this: https://github.com/trullock/NUglify/issues/245#issue-862625088

**Comment by trullock** [comment]
Its exactly the same as #245 so I'll close this as duplicate. You're not doing any literal templating in those regexes so just use `"` or `'` instead.

I'll try and take a look at #245 but from memory it wasnt simple, the way content is handled makes dealing with escapes difficult. Parts will need a rewrite
